lua/conquer.lua

```diff
@@ -115,22 +114,26 @@ end
 
 conquer.key_enter = function(og_bufnr)
   local lines = vim.api.nvim_buf_get_lines(0, 0, 1, false)
-  local results = vim.split(vim.fn.execute(lines), "\n")
+  local results = vim.fn.execute(lines)  -- Do not recalculate
+  local results_pretty = vim.split(vim.fn.execute(lines), "\n")
+
+  -- Copy into the '*' register
+  vim.api.nvim_command(string.format(':call setreg(\'*\', \'%s\')', results))
```

Comment:
Maybe a keymap to do this would be better. I don't know that you want to *always* copy it to your register.

Also, you should try using `vim.fn.setreg` instead of this method.

Reply:
Will think about the bindings and update some time today.